tests/Event/PersistenceEventTest.php

```diff
@@ -30,14 +30,17 @@ class PersistenceEventTest extends TestCase
     private $admin;
 
     /**
-     * @var mixed
+     * @var object
      */
     private $object;
 
     protected function setUp(): void
     {
-        $this->admin = $this->getMockForAbstractClass(AdminInterface::class);
-        $this->object = new \stdClass();
+        /** @var object $object */
```

Comment:
it does not detect stdClass as object?

Reply:
Psalm detect the stdClass as a stdClass and then throw
```
$this->event with declared type 'Sonata\AdminBundle\Event\PersistenceEvent<object>' cannot be assigned type 'Sonata\AdminBundle\Event\PersistenceEvent<stdClass>'
```